src/auditors/repository-actions-secrets.ts

```diff
@@ -9,16 +7,30 @@ export const auditor = async ({
   octokit,
   owner,
   repo,
-}: {
-  octokit: Octokit;
-  owner: string;
-  repo: string;
-}): Promise<AuditorWarning[]> => {
-  const { data } = await octokit.rest.actions.listRepoSecrets({
-    owner,
-    repo,
-    per_page: 1,
-  });
+  logger,
+}: AuditorFunctionArgs): Promise<AuditorWarning[]> => {
+  let data;
+
+  try {
+    const response = await octokit.rest.actions.listRepoSecrets({
+      owner,
+      repo,
+      per_page: 1,
+    });
+
+    data = response.data;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (error: any) {
```

Comment:
I don't like this, but I'm not aware of a better way to narrow types and know that this is a Octokit `ResponseError` 🤔 We can always come back and refactor this later.